core/pom.xml

```diff
@@ -156,6 +156,17 @@
             <classifier>models</classifier>
             <scope>provided</scope>
         </dependency>
+        <!-- Clustering attempt-->
+        <dependency>
+            <groupId>org.nield</groupId>
+            <artifactId>kotlin-statistics</artifactId>
+            <version>1.2.1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.edwardraff</groupId>
+            <artifactId>JSAT</artifactId>
+            <version>0.0.9</version>
+        </dependency>
 
```

Comment:
dependency versions must not be in submodules, but rather in root pom file, under dependencyManagment.
Recall what explained in `for_developers.md`.
You can pass by my office and I can explain in person the details of Maven if that is not clear :)